textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/SpanTuple.java

```diff
@@ -0,0 +1,39 @@
+package edu.uci.ics.textdb.common.field;
+
+import java.util.List;
+
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.IField;
+
+public class SpanTuple extends DataTuple{
+    
+    private IField spanField;
+    private Span span;
+    private String key;
+
+    public SpanTuple(List<Attribute> schema, IField[] fields, 

```

Comment:
@JavierJia .. I thought SpanTuple is something, which has the data tuple(which is nothing but the document) and also the occurrence of the key word. In the implementation you provided it only captures the occurrence of the given key word.